eng/scripts/automation/GenerateAndBuildLib.ps1

```diff
@@ -770,29 +786,44 @@ function GeneratePackage()
             $serviceProjFilePath = Join-Path $sdkRootPath 'eng' 'service.proj'
             dotnet build /p:Scope=$service /p:Project=$packageName /p:RunApiCompat=$false $serviceProjFilePath
             if ( !$? ) {
-                Write-Error "Failed to build sdk solution. exit code: $?"
+                Write-Host "[ERROR] Failed to build sdk solution:$packageName. exit code: $?."
+                Write-Host "[ERROR] Please review the detail errors for potential fixes. For guidance, visit https://aka.ms/azsdk/sdk-automation-faq ."
+                Write-Host "[ERROR] If the issue persists, contact the DotNet language support channel at $DotNetSupportChannelLink and include this spec pull request."
                 $result = "failed"
             }
             # pack
             Write-Host "Start to pack sdk"
             dotnet pack $srcPath /p:RunApiCompat=$false
             if ( !$? ) {
-                Write-Error "Failed to pack sdk. exit code: $?"
+                Write-Host "[ERROR] Failed to pack the following sdk with exit code: $?."
+                Write-Host "[ERROR] $srcPath."
```

Comment:
We should pass `$srcPath` to up line. It is strange that list a path alone in a line

Reply:
Added packageName and service to the output.